internal/driver/commands.go

```diff
@@ -189,6 +189,14 @@ var configHelp = map[string]string{
 		"Drops functions above the highest matched frame.",
 		"If set, all frames above the highest match are dropped from every sample.",
 		"Matching includes the function name, filename or object name."),
+	"tagroot": helpText(
+		"Adds psuedo-stack-frames for labels key/value pairs at the callstack root.",
+		"A comma-separated list of label keys.",
+		"The first key creates frames at the new root."),
```

Comment:
Could we clarify what happens with the samples that don't have the specified tag?

I think my expectation would be that:
* The levels corresponding to the specified tags will still be added
* No sample is dropped, instead they are attributed to a fake value like "foo:<unknown>" where "foo" is the tag name.

Reply:
As implemented now:
- a new frame corresponding to the specified tag is *not* added if that sample is missing a tag
- I'm not dropping any samples from the profile

But I think that adding the extra frame even if the tag is absent with something like `foo:` is reasonable. I'll code it up and see how it looks.

Reply:
I've updated the code to add the extra frame even if the tag doesn't match.